sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/unit/ChatCompletionsToolCallUnitTests.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.unit;
+
+import com.azure.ai.openai.models.ChatCompletionsFunctionToolCall;
+import com.azure.ai.openai.models.ChatCompletionsOptions;
+import com.azure.ai.openai.models.ChatCompletionsToolCall;
+import com.azure.ai.openai.models.ChatRequestAssistantMessage;
+import com.azure.ai.openai.models.FunctionCall;
+import com.azure.core.util.BinaryData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+public class ChatCompletionsToolCallUnitTests {
+
+    @Test
+    public void testSerialization() {
+        ChatCompletionsToolCall functionToolCall =
+            new ChatCompletionsFunctionToolCall("id", new FunctionCall("name", "arguments"));
+
+        ChatCompletionsOptions options = new ChatCompletionsOptions(Collections.singletonList(
+            new ChatRequestAssistantMessage("content").setToolCalls(Collections.singletonList(functionToolCall))));
+
+        String json = BinaryData.fromObject(options).toString();
```

Comment:
The behavior of serialization on the ChatCompletionsFunctionToolCall object is difference, between serialize `ChatCompletionsFunctionToolCall` instance directly to JSON, and serialize the `ChatCompletionsFunctionToolCall` that contains a `ChatCompletionsFunctionToolCall` to JSON.